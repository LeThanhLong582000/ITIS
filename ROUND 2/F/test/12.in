100
41389997 12469514 71841034
82156655 69189434 80540938
32991029 51888137 54527699
97259622 39020869 9866959
98524689 66950307 36558881
35766552 57142917 89853472
3193496 30511636 1421027
46115295 41078840 42758118
94195615 47368920 47499311
73387035 26102691 14781644
15524030 79660418 28802167
59217322 57801559 75161080
84828300 37543338 73167232
72178554 59670457 78096003
38697757 8341959 51623485
10945478 9391362 20772096
98823996 79612614 19529215
57651831 16191669 69127280
89976184 40520692 80285441
97425665 87214999 77893637
3067925 2040754 58495858
46912631 42474156 72843250
14147416 19069732 18955299
47734445 81324962 22651123
68704209 77493850 95781996
53064985 43040294 98065536
58753946 6205894 55722281
2730919 65541279 27437816
45646166 28748058 30082853
69022407 80032477 13179195
61164390 25020150 20181291
36276756 13249827 67131156
21856988 49563496 55818727
14305786 22634205 60767554
3791919 35874028 9798620
96263999 10380334 72510690
87393642 92952865 47542662
88023581 95712576 79911580
87318291 97175451 41549415
88920791 50474661 23661982
43240943 74231840 48546066
49079519 77427189 92292234
56407221 30273157 58293411
78713050 38945516 84612608
15735832 92810393 74124203
38555536 2503067 27480435
96701840 72896340 23002874
39397087 55686259 50045913
11128852 30106901 97710969
67908384 98370350 54091510
64029357 28157629 37036055
52740855 11789348 84009744
36460005 25903179 46936961
7256560 53631439 38443750
8754685 51065415 81175260
34502396 92334560 94820998
57393327 34940227 26221683
48919515 99904892 8547681
71629126 79841846 1791372
11178610 90065375 81622692
2251860 50881772 97328636
33998437 66932642 16015388
96208445 49826114 14719955
41002693 74159180 12582118
14213516 76576159 40969745
44345031 77405466 89050357
4714869 97625665 59346666
61995867 14852761 15402341
25718809 19033998 73572174
8646330 9968052 44375048
88950536 9307832 97727601
49419118 77862150 69231471
63227921 67234203 44370509
16203743 40133099 80695670
89756744 73395791 97667110
13352609 79895734 87963277
44939927 40434689 93256963
35349047 16857480 45202170
96603781 22304969 87855756
84171536 2287307 80037970
13855064 69507937 76950436
46585927 39783596 97039924
42973405 27069889 94405953
17160731 25846436 84346691
59963752 27452058 39119138
69047296 90882832 78030168
99834808 5391044 18229746
32875972 96694655 88302020
20934865 1981715 49923897
33808722 90551681 59997172
66652164 55213111 70809228
85401116 36614803 81420445
62753277 98971650 98342615
47609689 1914878 28148333
29243915 21938122 45087688
61729608 31721719 4318645
31056089 64900618 63988655
1345937 78110400 81409918
95462080 6462592 62114192
56684101 79238011 17571480
