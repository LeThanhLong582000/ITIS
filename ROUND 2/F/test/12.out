50380759
46754080
17815296
9866960
34066632
34238554
0
23897287
47163003
14781645
0
38288422
60226097
45302050
40989642
11163106
19370299
55293721
64870467
44052152
29761515
38640863
7016492
0
43496178
54045114
54135167
0
23490481
1084563
20181292
45079043
14056110
26219568
0
72510691
20991720
36111293
15846128
23661983
8777585
31972282
42213738
62190071
0
27480436
23002875
16878371
39366460
11814772
36453892
62480626
28746894
0
19432265
18494417
24337392
0
0
1367964
24349362
0
14719956
0
0
27994961
0
15402342
40128493
21526663
88685153
20394220
20182114
28383157
57014032
10710076
48881101
31846869
81077284
80037971
10648782
51921128
55154735
37830493
35815416
28097316
18229747
12241669
34438524
1627107
41124141
65103379
31062121
28148334
26196741
4318646
15072063
2322728
62114193
0
